### PR TITLE
Use registry.redhat.io to load server and operator

### PR DESCRIPTION
**Pull request by dmytro-ndp**
Fix default operator and server addresses to use **registry.redhat.io** instead of **quay.io** by default.

#### Issue to be fixed
https://issues.jboss.org/browse/CRW-506

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -194,7 +194,7 @@ OPTIONS
   --che-operator-cr-yaml=che-operator-cr-yaml  Path to a yaml file that defines a CheCluster used by the operator. This
                                                parameter is used only when the installer is the operator.
 
-  --che-operator-image=che-operator-image      [default: quay.io/crw/operator-rhel8:2.0] Container image of the
+  --che-operator-image=che-operator-image      [default: registry.redhat.io/codeready-workspaces/operator-rhel8:2.0] Container image of the
```
Comment (davidfestal) [inline README.md]: Isn't the name of this image `server-operator-rhel8` here ?

File: README.md [hunk 2]
```diff
@@ -265,7 +265,7 @@ OPTIONS
 
   -t, --templates=templates                [default: templates] Path to the templates folder
 
-  --che-operator-image=che-operator-image  [default: quay.io/crw/operator-rhel8:2.0] Container image of the operator.
+  --che-operator-image=che-operator-image  [default: registry.redhat.io/codeready-workspaces/operator-rhel8:2.0] Container image of the operator.
```
Comment (davidfestal) [inline README.md]: Isn't the name of this image `server-operator-rhel8` here ?

File: README.md (hunk 1 above)
Comment (dmytro-ndp) [reply README.md]: yes, my fault
Good catch!

File: README.md (hunk 2 above)
Comment (dmytro-ndp) [reply README.md]: Good catch! Thank you.

**Comment by dmytro-ndp** [comment]
@ibuziuk: in that case it's strange that there are other commits regarding README.md https://github.com/redhat-developer/codeready-workspaces-chectl/commit/a12cc1f233ff1807177286a8602ada3918c5bcd7#diff-04c6e90faac2675aa89e2176d2eec7d8

**Comment by ibuziuk** [comment]
@dmytro-ndp I mean someone need to run the command for update and than commit changes, but not fix the README manually. @benoitf could you please comment how this should be done?

**Comment by dmytro-ndp** [comment]
Meanwhile I have restored README.md.

**Comment by dmytro-ndp** [comment]
Thank you, @benoitf !

**Pull request merged**